rest/data/subscription_test.go

```diff
@@ -0,0 +1,72 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
is there a way to avoid this dependency?

Reply:
Yea, I don't actually need to initialise the ID.